packages/ethereum-light-client/src/client_state.rs

```diff
@@ -0,0 +1,39 @@
+use alloy_primitives::{Address, B256, U256};
+use serde::{Deserialize, Serialize};
+use serde_with::{serde_as, DisplayFromStr};
+
+use crate::types::{fork_parameters::ForkParameters, height::Height};
+
+#[serde_as]
+#[derive(Serialize, Deserialize, PartialEq, Clone, Debug, Default)]
+pub struct ClientState {
+    #[serde_as(as = "DisplayFromStr")]
```

Comment:
It is also because of the current client state from union that does this. But I do believe this is defined as a string in the etheruem specs, so that is probably why. Will also be looked into as part of #143